content/developers/api-reference/assets-api/index.md

```diff
@@ -332,16 +332,6 @@ curl -g -v -X GET \
      "https://app.datatrails.ai/archivist/v2/assets?proof_mechanism=MERKLE_LOG"
 ```
 
-#### Fetch Events Ordered for SIMPLEHASHV1 Schema
```

Comment:
We need to keep this: it's still pertinent to how we form Merkle Log leaves

Reply:
fixed